internal/provider/models/team.go

```diff
@@ -10,12 +10,13 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/types"
 )
 
-// Team describes the data source data model.
-type Team struct {
+// TeamDataSource describes the data source data model.
+type TeamDataSource struct {
 	Id               types.String `tfsdk:"id"`
 	Name             types.String `tfsdk:"name"`
 	Description      types.String `tfsdk:"description"`
 	IsIdpManaged     types.Bool   `tfsdk:"is_idp_managed"`
+	OrganizationId   types.String `tfsdk:"organization_id"`
```

Comment:
discovered we were actually missing `OrganizationId` and added this field here and to schema and tests

Reply:
i purposely removed it since its clear that the organizationId will be the same one that is configured in the provider config

Reply:
ah... ok